meta-genivi-dev/recipes-dev-platform/images/genivi-dev-platform.bb

```diff
@@ -32,6 +32,7 @@ IMAGE_INSTALL_append = " \
     packagegroup-gdp-qt5 \
     packagegroup-gdp-rvi \
     packagegroup-gdp-dev \
+    chromium-wayland \
```

Comment:
I mean if you don't have Chromium included in the image, wouldn't that mean HMI will have a button to launch an app that doesn't exist? I might be misunderstanding something here though.